CollectD/Page_PostgreSQL.json

```diff
@@ -5,8 +5,8 @@
     {
         "marshallId": 1,
         "sf_type": "Service",
-        "sf_description": "Postgresql metrics.", 
-        "sf_service": "Postgresql"
+        "sf_description": "Metrics about PostgreSQL.", 
+        "sf_service": "PostgreSQL"
```

Comment:
We can. On a similar note I updated the service name for "AWS OpsWorks" from its current "AWS Oopsworks", do you think I should remove that correction as well?